frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmInstallUpgradePage.tsx

```diff
@@ -55,9 +56,10 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
 
   const namespace = match.params.ns || searchParams.get('preselected-ns');
   const initialChartURL = decodeURIComponent(searchParams.get('chartURL'));
+  const indexEntry = decodeURIComponent(searchParams.get('indexEntry'));
   const initialReleaseName = match.params.releaseName || '';
-  const helmChartName = searchParams.get('chartName');
-  const helmChartRepoName = searchParams.get('chartRepoName');
+  const helmChartName = decodeURIComponent(searchParams.get('chartName'));
+  const helmChartRepoName = decodeURIComponent(searchParams.get('chartRepoName'));
```

Comment:
Same here. `encodeURIComponent` and `decodeURIComponent` is only needed when you build and extract something manually from an URL. When we use URLSearchParams this is not needed.

```suggestion
  const helmChartName = searchParams.get('chartName');
  const helmChartRepoName = searchParams.get('chartRepoName');
```